pom.xml

```diff
@@ -71,7 +71,7 @@
         <dependency>
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
-            <version>[4.3.0,)</version>
+            <version>4.7.0</version>
```

Comment:
we still want to pick up most recent version possible, not exact one

Reply:
fixed